lib/client.js

```diff
@@ -111,7 +111,7 @@ function listen(server, portRange) {
 
 function padId(id) {
   
-  var newId = new Buffer(20);
+  var newId = new Buffer.alloc(20);
```

Comment:
This should remove the `new` keyword since `alloc` is a static function?

```suggestion
  var newId = Buffer.alloc(20);
```

Reply:
you're right. I was overlooked.
will send another PR